packages/flutter_tools/lib/flutter_tools.dart

```diff
@@ -12,7 +11,7 @@ import 'src/flx.dart' as flx;
 /// pre-compiled snapshot.
 Future<int> assembleFlx({
   Map manifestDescriptor: const {},
-  ArchiveFile snapshotFile: null,
+  File snapshotFile: null,
```

Comment:
@tvolkert - I forget to cc you on this. This will change the public API of the package. Hopefully not too much work to update the code using this.